core/model/modx/processors/resource/update.class.php

```diff
@@ -452,6 +452,18 @@ public function checkForUnPublishOnSiteStart() {
      * @return boolean
      */
     public function checkDeletedStatus() {
+	    if (1 == $this->getProperty('deleted', null)) {
+	    	if ($this->modx->getOption('site_start') == $this->getProperty('id')) {
```

Comment:
This won't work in a multi-context setup. I guess it would be better to use `$this->workingContext->getOption()` instead